README.rst

```diff
@@ -25,6 +25,28 @@ found `here <http://mushorg.github.io/conpot/>`_. There you will also find the i
 `install <http://mushorg.github.io/conpot/installation/ubuntu.html>`_ conpot and the 
 `FAQ <http://mushorg.github.io/conpot/faq.html>`_.
 
+EASY INSTALL USING DOCKER CONTAINER
+-----------------------------------
+
+**Via a pre-built image**
+
+1. Install `Docker <https://www.docker.com/>`_
+2. Run ``docker pull honeynet/conpot``
+3. Run ``docker run -it -p 80:80 -p 102:102 -p 502:502 -p 161:161/udp --network=bridge honeynet/conpot:latest /bin/sh``
+4. Finally run ``conpot --template default``
+
+Navigate to ``http://MY_IP_ADDRESS`` to confirm the setup. **Or, you may also** 
+
+**Build docker image from source**
+
+1. Install `Docker <https://www.docker.com/>`_
+2. Download the Dockerfile from conpot repo ``wget https://raw.githubusercontent.com/mushorg/conpot/master/Dockerfile``
+3. Run ``docker build -t conpot .``
+4. Run ``docker run -it -p 80:80 -p 102:102 -p 502:502 -p 161:161/udp --network=bridge conpot:latest /bin/sh``
+5. Finally run ``conpot --template default``
+
+Navigate to ``http://MY_IP_ADDRESS`` to confirm the setup. 
```

Comment:
Like I installed conpot on an Ubuntu VM via docker.  When navigate to the IP of the VM, I am greeted with this message.. Because of the service running on port 80.
![image](https://cloud.githubusercontent.com/assets/4391664/22544997/0c9d0770-e95d-11e6-8269-57cc4050ca90.png)